tests/php/Extension/Engine/SiteTreePublishingEngineTest.yml

```
@@ -0,0 +1,24 @@
+SilverStripe\CMS\Model\SiteTree:
+  page1:
+    Title: Page 1
+    URLSegment: page-1
+  page2:
+    Title: Page 2
+    URLSegment: page-2
+    ParentID: =>SilverStripe\CMS\Model\SiteTree.page1
```

```suggestion
    Parent: =>SilverStripe\CMS\Model\SiteTree.page1
```
[nitpick] Prefer to use relation name rather than foreign key. Applies to multiple instances of this change-set.